### PR TITLE
Update learner Gherkins for 0.13

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary
Ready for review & merge!

### Reviewer guidance
…

### References
…

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: integration_testing/features/learner/learner-engage-content-channels.feature [hunk 1]
```diff
@@ -18,13 +18,17 @@ Feature: Learner engages content channels
...
+    Then I see the <content_item> page
       And I see the video start playing automatically
       # bug in Firefox, issue reported
       And I see the *Next resource* heading and content item under the player
     When video finishes
-    Then I see the *Next* snackbar alert with the title of the next recommended content item
-      And I see the *+500 points* snackbar alert
-      And I see my points counter is increased by 500 
+    Then I see the status icon is yellow star
+    # Then I see the *Next* snackbar alert with the title of the next recommended content item
```
Comment (radinamatic) [inline integration_testing/features/learner/learner-engage-content-channels.feature]: @khangmach Please confirm that the snackbars alerts for `next resource` (above) and points (see the comment below, line 30) were removed intentionally in 0.13

File: integration_testing/features/learner/learner-engage-content-recommended.feature [hunk 2]
```diff
@@ -7,20 +7,20 @@ Feature: Learner engages recommended content
       And I am on *Learn > Recommended* page
 
   Scenario: Engage with content on Recommended tab
-    When I see content items under the *Most popular*, *Next steps* and *Resume* headings
+    When I see content items under the *Most popular* and *Recommended* headings
+    # *Next steps* and *Resume* ???
```
Comment (radinamatic) [inline integration_testing/features/learner/learner-engage-content-recommended.feature]: @khangmach Please confirm that the *Next steps* and *Resume* sections were removed intentionally from the **Recommended** tab in 0.13.

File: integration_testing/features/learner/learner-engage-content-recommended.feature [hunk 3]
```diff
@@ -7,20 +7,20 @@ Feature: Learner engages recommended content
...
     Then I can use the right arrow buttons to see more in each section
       And after that I can use the left arrow buttons to browse back
     When I click the <content_item> content item
-    Then I see the *Recommended > '<content_item>'* breadcrumb
+    Then I see the *Recommended > '<content_item>'* page
       And I see the video start playing automatically
-      And below the player I see other *Recommended* items
+    #  And below the player I see other *Recommended* items
```
Comment (radinamatic) [inline integration_testing/features/learner/learner-engage-content-recommended.feature]: @khangmach Please confirm that other recommended items below the currently viewed resource were removed intentionally in 0.13.

File: integration_testing/features/learner/learner-engage-content-channels.feature (hunk 1 above)
Comment (khangmach) [reply integration_testing/features/learner/learner-engage-content-channels.feature]: I was not made aware of this removal. Not intentional as far as I'm aware.

File: integration_testing/features/learner/learner-engage-content-recommended.feature (hunk 3 above)
Comment (khangmach) [reply integration_testing/features/learner/learner-engage-content-recommended.feature]: Don't recall concrete decisions on this. Maybe @indirectlylit or @jtamiace can shed some light on this

File: integration_testing/features/learner/learner-change-profile-information.feature [hunk 4]
```diff
@@ -3,24 +3,36 @@ Feature: Learner changes profile information
 
   Background:
     Given I am signed in to Kolibri as a learner user
-      And I am on the *'<username>' > Profile* page
-      And I have permission to edit my full name and username
+      And I am on my <username> *Profile > Detils* page
```
Comment (mrpau-richard) [inline integration_testing/features/learner/learner-change-profile-information.feature]: Typo error here @radinamatic 
*Profile > Detils*

File: integration_testing/features/learner/learner-change-profile-information.feature [hunk 5]
```diff
@@ -3,24 +3,36 @@ Feature: Learner changes profile information
...
+      And facility is set up to allow learners and coaches to edit full names and usernames
 
   Scenario: Learner changes username and full name
-  	When I change my full name
+  	When I clic the *Edit* button
```
Comment (mrpau-richard) [inline integration_testing/features/learner/learner-change-profile-information.feature]: Typo error `clic`

File: integration_testing/features/learner/learner-change-profile-information.feature [hunk 6]
```diff
@@ -3,24 +3,36 @@ Feature: Learner changes profile information
...
+    Then I see the *Edit profile* page
+    When I change my full name
   	  And I change my username
   	  And if my changes are valid (I did not leave the fields empty)
-  	  And I click the “Save changes” button
-    Then I see the *Profile details updated* inline notification
-      And I see the new full name and username on the profile page
+  	  And I click the “Save” button
+    Then I see the *Profile details updated* snackbar notification
+    When I go back to my *Profile > Detils* page
```
Comment (mrpau-richard) [inline integration_testing/features/learner/learner-change-profile-information.feature]: Typo error `Detils`

File: integration_testing/features/learner/learner-change-profile-information.feature [hunk 7]
```diff
@@ -3,24 +3,36 @@ Feature: Learner changes profile information
...
+    Then I see the new full name and username on the profile page
       And I see the new username in the user menu
 
+    Scenario: Learner selects gender and birth year
+    When I clic the *Edit* button
```
Comment (mrpau-richard) [inline integration_testing/features/learner/learner-change-profile-information.feature]: Typo error `clic`

**Review by mrpau-richard — APPROVED** [review]

**Pull request merged**